internal/common/builtin_structs.rs

```diff
@@ -90,6 +90,8 @@ macro_rules! for_each_builtin_structs {
                     text: SharedString,
                     /// The keyboard modifiers active at the time of the key press event.
                     modifiers: KeyboardModifiers,
+                    /// Whether or not this event is for a key press repeat event.
+                    repeat: bool,
```

Comment:
:+1:

Reply:
I’m curious: why is the field called repeat instead of repeated?

Reply:
That was just to be consistent with:
- https://w3c.github.io/uievents/#dom-keyboardevent-repeat
- https://docs.rs/winit/0.29.3/winit/event/struct.KeyEvent.html#structfield.repeat

Trying to understand their reason I guess "repeat" can state it less ambiguous that "this is a repeat event" while "this is a repeated event" could mean that it's been repeated. e.g. the first event is "repeated" but not a "repeat".

I spent some time digging out some discussions and the earliest I could find is this thread where they seem to try to express this as a distinct "key repeat event" and not a "key press that is repeated": https://lists.w3.org/Archives/Public/www-dom/2005JanMar/0003.html

But I'm not attached to it either way, that part is up to you :)

Reply:
Thank you for the very detailed explanation. I agree with your choice.